example-fragments/otp-fragment/src/client/react.ts

```
@@ -0,0 +1,8 @@
+import type { FragnoPublicClientConfig } from "@fragno-dev/core/client";
+import { useFragno } from "@fragno-dev/core/react";
+
+import { createOtpFragmentClients } from "..";
+
+export function createOtpFragmentClient(config: FragnoPublicClientConfig = {}) {
+  return useFragno(createOtpFragmentClients(config));
```

Biome (2.4.6)

[error] This hook is being called from within a function or method that is not a hook or component.

(lint/correctness/useHookAtTopLevel)